### ISSUE TITLE
One travis test failing on LIBSVM model OneClass

### ISSUE THREAD

**Issue by ablaom** [alan-turing-institute/MLJModels.jl#21]
Tolerance for test too small? 

```
LIBSVM: Test Failed at /Users/travis/build/alan-turing-institute/MLJModels.jl/test/LIBSVM.jl:89
  Expression: isapprox((length(train) - sum(MLJBase.transform(oneclasssvm, fitresultoc, selectrows(X, train)) .== true)) / length(train), oneclasssvm.nu, atol=0.001)
   Evaluated: isapprox(0.10166666666666667, 0.1; atol=0.001)
Stacktrace:
 [1] top-level scope at none:0
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1044
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] include(::String) at ./client.jl:392
 [6] macro expansion at /Users/travis/build/alan-turing-institute/MLJModels.jl/test/runtests.jl:37 [inlined]
 [7] macro expansion at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
 [8] top-level scope at /Users/travis/build/alan-turing-institute/MLJModels.jl/test/runtests.jl:37
```

**Comment by ablaom** [comment]
Ah. We are also using random data for the test.

**Comment by ablaom** [comment]
Resolved  b3eb20900601d5a5f36ddba18e2db401bbf559c4
